java-checks-test-sources/src/main/java/checks/StringLiteralDuplicatedCheck.java

```diff
@@ -36,22 +35,36 @@ default void foo() {
   private static final String REPORT_WITHOUT_THRESHOLD = "blabla";
 
   void test() {
-    System.out.println("constant"); // Noncompliant [[secondary=40]] {{Use already-defined constant 'A' instead of duplicating its value here.}}
+    System.out.println("constant"); // Noncompliant [[secondary=39]] {{Use already-defined constant 'A' instead of duplicating its value here.}}
```

Comment:
We can use relative line numbers for secondaries now (`secondary=+1`), so we should do that whenever updating a secondary location, so it's the last time we have to update it :wink: 

The same applies to the secondary locations in the other test files.